src/app/modules/webhook/webhook.service.ts

```diff
@@ -243,21 +251,29 @@ export const createInitialWebhookSender =
   > => {
     // Attempt to send webhook
     return sendWebhook(submission.getWebhookView(), webhookUrl).andThen(
-      (webhookResponse) =>
+      (webhookResponse) => {
+        webhookStatsdClient.increment('sent', 1, 1, {
+          responseCode: webhookResponse.response.status.toString(),
```

Comment:
⚠️ Is there a risk of runtime error here?

Can I confirm that network errors (cases where `response.status` is `undefined`!) do NOT enter this handler? And if that is the case, do we report network errors in the metric?

A simple fix can be to remove the `.toString()` here, or add a safety check:
```typescript
responseCode: webhookResponse.response.status?.toString() || null
```

Reply:
I've changed this to a template literal instead!